src/controllers/dynakube/activegate/internal/authtoken/config.go

```diff
@@ -5,5 +5,5 @@ import (
 )
 
 var (
-	log = logger.NewDTLogger().WithName("dynakube-activegate-authtoken")
+	log = logger.Factory.GetLogger("dynakube-activegate-authtoken")
```

Comment:
```suggestion
	log = logger.Factory.GetLogger("dynakube.activegate.authtoken")
```
or we could just leave out `dynakube` since it's implied

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/1223#issuecomment-1274464902